docs/content/how-to/cluster-configuration.md

```diff
@@ -0,0 +1,104 @@
+# Cluster Configuration
```

Comment:
Since this is a how-to, maybe change to an active voice? e.g. "Configure OCP components"